client/components/welcome/index.tsx

```diff
@@ -13,27 +13,27 @@ import wooPaymentsLogo from 'assets/images/woopayments.svg?asset';
 import './style.scss';
 
 /**
- * Renders the header of the account's balances card.
+ * Renders a welcome card header with a greeting and the Woo Payments logo.
```

Comment:
This will ultimately also display important notices, right?

Reply:
Thanks! I've added this detail in 41ae333

Reply:
**Update**

@haszari I've reverted this description to clarify that this component only returns the card header (greeting + logo). 7c66faf